common/processor-utils/src/main/kotlin/br/com/zup/beagle/compiler/RegisteredActionGenerator.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2020 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package br.com.zup.beagle.compiler
+
+import br.com.zup.beagle.annotation.RegisterAction
+import com.squareup.kotlinpoet.ClassName
+import com.squareup.kotlinpoet.FunSpec
+import com.squareup.kotlinpoet.KModifier
+import com.squareup.kotlinpoet.ParameterizedTypeName.Companion.parameterizedBy
+import com.squareup.kotlinpoet.asClassName
+import javax.annotation.processing.RoundEnvironment
+
+class RegisteredActionGenerator {
+
+    fun generate(roundEnvironment: RoundEnvironment): FunSpec {
+        val classValues = StringBuilder()
+        val registerAnnotatedClasses = roundEnvironment.getElementsAnnotatedWith(RegisterAction::class.java)
+        val listReturnType = List::class.asClassName().parameterizedBy(
+            Class::class.asClassName().parameterizedBy(
+                ClassName(ANDROID_ACTION.packageName, ANDROID_ACTION.className)
+            )
+        )
+
+        registerAnnotatedClasses.forEachIndexed { index, element ->
+            classValues.append("\t${element}Binding::class.java as Class<Action>")
+            if (index < registerAnnotatedClasses.size - 1) {
+                classValues.append(",\n")
+            }
+        }
```

Comment:
Why not just `joinToString`?
```suggestion
        registerAnnotatedClasses.joinToString(",\n") { element ->
            "\t${element}Binding::class.java as Class<Action>"
        }
```